codecombat/codecombat
# Level UX PlayLevelView base changes

# Why?

We are beginning work on level UX and need a structure we can change and update in the play level view.

# How? 

All necessary pieces have been moved into the Ozaria folder, and all references updated, so we can freely make changes moving forwards. 

# How to review/test

Don't be alarmed by the 126 changed files. It's actually just this change: https://github.com/codecombat/codecombat/pull/5433/files#diff-5a00df3717cc9ade14619a5f033f1c9bL37

The rest are files moved from the regular part of the codebase to /ozaria

For testing, you can check out this branch locally and go to play a level under the ozaria path. Things should work normally.
